### PR TITLE
javax.enterprise => jakarta.enterprise

### PR DESCRIPTION
**Pull request by scottmarlow**
Signed-off-by: Scott Marlow <smarlow@redhat.com>

### CODE REVIEW

File: src/com/sun/ts/tests/jaxrs/platform/managedbean299/TSAppConfig.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2020 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2018 Oracle and/or its affiliates. All rights reserved.
```
Comment (alwin-joseph) [inline src/com/sun/ts/tests/jaxrs/platform/managedbean299/TSAppConfig.java:2]: Shouldn't this be 2020 ?

File: src/com/sun/ts/tests/jaxrs/platform/managedbean299/TSAppConfig.java [hunk 2]
```diff
@@ -19,11 +19,11 @@
 import java.util.HashSet;
 import java.util.Set;
 
-import jakarta.annotation.PostConstruct;
-import javax.enterprise.context.ApplicationScoped;
-import jakarta.ws.rs.core.Application;
-import jakarta.ws.rs.core.Context;
-import jakarta.ws.rs.core.UriInfo;
+import javax.annotation.PostConstruct;
```
Comment (alwin-joseph) [inline src/com/sun/ts/tests/jaxrs/platform/managedbean299/TSAppConfig.java:22]: jakarta.annotation and jakarta.ws were reverted to resolve conflicts ?

File: src/com/sun/ts/tests/jaxrs/platform/managedbean299/TSAppConfig.java (hunk 1 above)
Comment (scottmarlow) [reply src/com/sun/ts/tests/jaxrs/platform/managedbean299/TSAppConfig.java:2]: yes, thanks!  Fixing

File: src/com/sun/ts/tests/jaxrs/platform/managedbean299/TSAppConfig.java (hunk 2 above)
Comment (scottmarlow) [reply src/com/sun/ts/tests/jaxrs/platform/managedbean299/TSAppConfig.java:22]: thanks, will figure out what happened and fix

Comment (scottmarlow) [reply src/com/sun/ts/tests/jaxrs/platform/managedbean299/TSAppConfig.java:22]: yes, I did that when I was resolving the 3 file conflicts, oops, fixing...

**Pull request merged**